packages/contracts-bedrock/scripts/upgrades/v1.8.0-permissioned/scripts/common.sh

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Associative array to store cached TOML content for different URLs
+# Used by fetch_standard_address and fetch_superchain_config_address
+declare -A CACHED_TOML_CONTENT
+
+export CONTRACTS_VERSION=${CONTRACTS_VERSION:-1.8.0-rc.3}
```

Comment:
Will need to adjust this to the final version once it's available and check if we need different versions for sepolia and mainnet.